includes/polling/os/eaton-mgeups.inc.php

```diff
@@ -7,5 +7,5 @@
 
```

Comment:
Please convert this to the new os format, https://docs.librenms.org/Developing/os/Initial-Detection/#os-discovery

Reply:
done